modules/deployment/pom.xml

```diff
@@ -8,6 +8,10 @@
         <version>1.0.0-SNAPSHOT</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
+    <properties>
+        <openapiGenerator.skip>true</openapiGenerator.skip>
```

Comment:
@jinyangyang222 can you please rename this to `generate.terraform.boot.client` and update the readme also accordingly.

Reply:
fixed